### PR TITLE
Cold War assets mod v1.0 support

### PR DESCRIPTION
**Pull request by MetalStormGhost**
Implemented support for the Cold War assets mod (ex Tu-16) v1.0 by tripod3

### CODE REVIEW

File: game/factions/faction.py [hunk 1]
```diff
@@ -543,6 +543,8 @@ def apply_mod_settings(self, mod_settings: Optional[ModSettings] = None) -> None
             self.remove_preset("LvS-103 Rb103A Mobile")
             self.remove_preset("LvS-103 Rb103B")
             self.remove_preset("LvS-103 Rb103B Mobile")
+        if not mod_settings.coldwarassets:
+            self.remove_aircraft("EA_6B")
```
Comment (Raffson) [inline game/factions/faction.py]: This looks a bit odd... care to explain? 😅

I'm assuming it's a fix for something that was forgotten when introducing support for the EA-6B?
Also, I'm expecting the B_47, Tu-16, etc. in this file 😅

File: changelog.md [hunk 2]
```diff
@@ -8,6 +8,8 @@
 * **[Squadrons]** Ability to define a livery-set for each squadron from which Retribution will randomly choose during mission generation
 * **[Modding]** Updated support for F/A-18E/F/G mod version 2.2.5
 * **[Modding]** Added VSN F-106 Delta Dart mod support (v2.9.4.101)
+* **[Modding]** Added VSN EA-6B Prowler mod support (v2.9.4.102)
```
Comment (Raffson) [inline changelog.md:11]: Missing changelog entry?

File: resources/units/aircraft/tu_22KD.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+tasks:
+  Anti-ship: 100
+  BAI: 350
+  CAS: 350
+  DEAD: 150
+  OCA/Aircraft: 350
+  OCA/Runway: 600
+  Strike: 600
+  SEAD: 30
+  SEAD Escort: 30
```
Comment (Raffson) [inline resources/units/aircraft/tu_22KD.yaml]: A bomber as SEAD Escort? Perhaps SEAD Sweep is a better choice here? 😅

File: resources/units/aircraft/B_47.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+max_range: 2000
+variants:
+  B-47 Stratojet: {}
+tasks:
+  Anti-ship: 90
+  DEAD: 200
+  OCA/Runway: 610
+  Strike: 610
+  SEAD: 90
+  SEAD Escort: 90
```
Comment (Raffson) [inline resources/units/aircraft/B_47.yaml]: Lol, I only noticed this after noticing the TU22KD 😅

**Review by Raffson — CHANGES_REQUESTED** [review]

File: game/factions/faction.py (hunk 1 above)
Comment (MetalStormGhost) [reply game/factions/faction.py]: Oops! Forgot to do something here 😬

Comment (MetalStormGhost) [reply game/factions/faction.py]: This has now been done.

File: changelog.md (hunk 2 above)
Comment (MetalStormGhost) [reply changelog.md:11]: I don't remember anymore, but it's possible I forgot to add a changelog entry for the EA-6B Prowler mod originally, so I might've included that in this PR.

File: resources/units/aircraft/tu_22KD.yaml (hunk 3 above)
Comment (MetalStormGhost) [reply resources/units/aircraft/tu_22KD.yaml]: Agreed, done.

File: resources/units/aircraft/B_47.yaml (hunk 4 above)
Comment (MetalStormGhost) [reply resources/units/aircraft/B_47.yaml]: Agreed, done.

**Comment by Raffson** [comment]
Seems like there's a merge-conflict, probably a result of the merge of the SuperBug?

**Review by Raffson — APPROVED** [review]

**Pull request merged**